src/components/header.js

```diff
@@ -1,23 +1,112 @@
-import { Link } from "gatsby"
+import { Link as GatsbyLink } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
-import { Box, Text } from "rebass"
+import { Box, Image, Flex, Link } from "rebass"
+import Logo from "../images/logo.svg"
+import Container from "./container"
 
 const Header = ({ siteTitle }) => (
-  <Box as="header" mb={2} bg="darkBlue">
-    <Box m="0 auto" maxWidth={960} px={1} py={2}>
-      <Text as="h1" fontSize={5} variant="heading">
-        <Link
+  <Box mb={0} as="header">
+    <Container>
+      <Flex color="white" alignItems="center" py={0}>
```

Comment:
Please use `justifyContent='space-between'` if you are spacing 2 sections in the Flex container so you can push the logo the left and nav wrapper to the right;